hack/verify-api-reference-docs.sh

```diff
@@ -26,11 +26,16 @@ source "${KUBE_ROOT}/hack/lib/init.sh"
 kube::golang::setup_env
 
 API_REFERENCE_DOCS_ROOT="${KUBE_ROOT}/docs/api-reference"
+# We want to pass absolute dir to update-api-reference-docs.sh, but diff
+# against local path to dir. This only matters in the case where this
+# test is itself running inside a container (with the host docker mapped in).
+REPO_DIR=${REPO_DIR:-"${PWD}/${KUBE_ROOT}"}
```

Comment:
I thought ${KUBE_ROOT} will become an absolute path after running `source "${KUBE_ROOT}/hack/lib/init.sh"`, I guess this is not the case if you are running the script inside a container?